tests/functional/tests/management/test_start_stop.py

```diff
@@ -330,6 +330,31 @@ def test_start_cache_same_id(pyocf_ctx, mode, cls):
     cache.get_stats()
 
 
+@pytest.mark.parametrize("cls", CacheLineSize)
+def test_start_cache_huge_device(pyocf_ctx_log_buffer, cls):
+    """
+    Test whether we can start cache which would overflow ocf_cache_line_t type.
+    pass_criteria:
+      - Starting cache on device too big to handle should fail
+    """
+    class HugeDevice(Volume):
+        def get_length(self):
+            return Size.from_MiB(353888999)
```

Comment:
It would be better to move this test to pyocf, where we can start cache with `metadata_volatile = true`. That way we can make it independent of metadata size.

Reply:
Thanks! I think it's fixed now